src/query/storages/fuse/src/table_functions/infer_schema/infer_schema_table.rs

```diff
@@ -57,8 +56,8 @@ const INFER_SCHEMA: &str = "infer_schema";
 
 pub struct InferSchemaTable {
     table_info: TableInfo,
-    location: String,
-    files_info: StageFilesInfo,
+    args_parsed: InferSchemaArgsParsed,
+    table_args: TableArgs,
```

Comment:
So we will defer the parse logic later?

Reply:
currently, TableArgs is reconstructed later (can be refactored later), 
so I keep a copy to simplify it.